manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.6.yaml

```diff
@@ -594,7 +594,7 @@ spec:
               config:
                 configuration: |
                   {
-                    "log_window_duration": {{ .Values.telemetry.v2.accessLogPolicy.logWindowDuration | quote }}
+                    "log_window_duration": "{{ .Values.telemetry.v2.accessLogPolicy.logWindowDuration }}"
```

Comment:
why not use quote function?

Reply:
it didn't work.. when I inspected envoyfilter for 1.6 too, quotes weren't there..